.travis.yml

```diff
@@ -37,14 +37,12 @@ perl:
   - "5.8.8-thr"
   - "5.8.9"
   - "5.8.9-thr"
-  - "5.8"
```

Comment:
This is official travis version of perl which is used by other travis jobs. I suggest to have it there for complete testing...

Reply:
was wondering indeed if one was the travis build and the other the build on demand let's readd both with a comments :-)